tasks/main.yml

```diff
@@ -8,9 +8,17 @@
   unarchive: src=/usr/local/src/pgweb_{{ pgweb_version }}_{{ pgweb_os }}_{{ pgweb_arch }}.zip
              dest=/usr/local/bin
              copy=no
-             creates=/usr/local/bin/pgweb
+             creates=/usr/local/bin/pgweb_{{ pgweb_version }}_{{ pgweb_os }}_{{ pgweb_arch }}
```

Comment:
It does, but then the next step renames to match this.  I wanted to skip the unarchive if the file was already present, but I wasn't sure how to unarchive to a different file name than what is present in the zip